storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst

```diff
@@ -0,0 +1,77 @@
+.. xbcloud_azure:
+
+=============================================================
+Using the xbcloud binary with Microsoft Azure Cloud Storage
+=============================================================
+
+This feature is *technical preview* quality.
+
+Implemented in Percona XtraBackup 8.0.27-19, the **xbcloud** binary adds support for the Microsoft Azure Cloud Storage using the REST API. 
+
+Options 
+----------------
+
+The following are the options, environment variables, and descriptions for uploading a backup to Azure using the REST API. The environment variables are recognized by **xbcloud**, which maps them automatically to the corresponding parameters:
+
+.. list-table::
+   :widths: 40 40 40
+   :header-rows: 1
+
+   * - Option name
+     - Environment variables
+     - Description
+   * - ``--azure-storage-account=name``
+     - AZURE_STORAGE_ACCOUNT
+     - An Azure storage account is a unique namespace to access and store your Azure data objects.
+   * - ``--azure-container-name=name``
+     - AZURE_CONTAINER_NAME
+     - A container name is a valid DNS name that conforms to the `Azure naming rules <https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names>`__
+   * - ``--azure-access-key=name``
+     - AZURE_ACCESS_KEY
+     - A generated key that can be used to authorize access to data in your account using the Shared Key authorization.
+   * - ``--azure-endpoint=name``
+     - AZURE_ENDPOINT
+     - The endpoint allows clients to securely access data
+   * - ``--azure-tier-class=name``
+     - AZURE_STORAGE_CLASS
+     - Cloud tier can decrease the local storage required while maintaining the performance. When enabled, this feature has the following categories:
+
+       - Hot - Frequently accessed or modified data
+
+       - Cool - Infrequently accessed or modified data
+
+       - Archive - Rarely accessed or modified data 
+
+With the `Azurite open-source emulator <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio>`__ you test Azure applications. For testing purposes, the **xbcloud** binary adds the ``--azure-development-storage`` option which uses the default ``access_key`` and ``storage account`` of azurite and ``testcontainer`` for the container. You can be overwrite these options, if needed.
```

Comment:
For testing purposes, the **xbcloud** binary adds the ``--azure-development-storage`` option, which uses the default ``access_key`` and ``storage account`` of azurite and ``testcontainer`` for the container. You can overwrite these options if needed.

Reply:
Thanks.